content/onboarding.rst

```diff
@@ -129,10 +133,11 @@ The initial stage is over when all the Onboarding tickets from the milestone ded
 
 We are far from single-sign-on and while working on this project you will have many different accounts.
 
-For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations (Chevah and ProAtria) and our core team. Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
+For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations and our core team.
+Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
```

Comment:
This should be removed, as this is a ProAtria only requirement and I hope that we can soon get rid of it.